modules/pubmaticAnalyticsAdapter.js

```diff
@@ -429,7 +429,7 @@ function getFloorFetchStatus(floorData) {
 function getListOfIdentityPartners() {
   const namespace = getGlobal();
   const publisherProvidedEids = namespace.getConfig("ortb2.user.eids") || [];
-  const availableUserIds = namespace.adUnits[0]?.bids[0]?.userId || {};
+  const availableUserIds = namespace.getUserIds() || {};
   const identityModules = namespace.getConfig('userSync')?.userIds || [];
```

Comment:
On line 54 above there is a browser map that is duplicated in the rtd adapter. Solve this and any other duplication before making further changes to either adapter